bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/VBS/WWjj_SS_dim8/WWjj_SS_dim8_ewk_customizecards.dat

```diff
@@ -0,0 +1,20 @@
+set param_card anoinputs 1 0.0
+set param_card anoinputs 2 0.0
+set param_card anoinputs 3 0.0
+set param_card anoinputs 4 0.0
+set param_card anoinputs 5 0.0
+set param_card anoinputs 6 0.0
+set param_card anoinputs 7 0.0
+set param_card anoinputs 8 0.0
+set param_card anoinputs 9 0.0
+set param_card anoinputs 10 0.0
+set param_card anoinputs 11 0.0
+set param_card anoinputs 12 2.000000e-12
```

Comment:
why not 0.0?

Reply:
because this was the optimal point to start from in term of statistic I think. Then the SM is recovered as one of the LHE weight here:
https://github.com/xjanssen/genproductions/blob/master/bin/MadGraph5_aMCatNLO/cards/production/pre2017/13TeV/VBS/WWjj_SS_dim8/WWjj_SS_dim8_ewk_reweight_card.dat#L75